### ISSUE TITLE
TypeError: import_ir_module(): incompatible function arguments.

### ISSUE THREAD

**Issue by Tagussan** [aws/aws-neuron-sdk#375]
Hi,

I have compiled a model in a g4dn instance and going to deploy it on inf1 instance. Although the compilation was successfully finished, I met the following error when running inference on inf1 instance
```
../anaconda3/envs/aws_neuron_pytorch_p36/lib/python3.6/site-packages/torch_neuron/jit_load_wrapper.py:13: in wrapper
    script_module = jit_load(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

f = PosixPath('******') 
map_location = None, _extra_files = ExtraFilesMap{}

    def load(f, map_location=None, _extra_files=DEFAULT_EXTRA_FILES_MAP):
        r"""
            Load a :class:`ScriptModule` or :class:`ScriptFunction` previously
            saved with :func:`torch.jit.save <torch.jit.save>`

.... long lines of comments omitted ....

        if isinstance(f, string_classes):
            if not os.path.exists(f):
                raise ValueError("The provided filename {} does not exist".format(f))
            if os.path.isdir(f):
                raise ValueError("The provided filename {} is a directory".format(f))
        if isinstance(map_location, string_classes):
            map_location = torch.device(map_location)
        elif not (map_location is None or
                  isinstance(map_location, torch.device)):
            raise ValueError("map_location should be either None, string or torch.device, "
                             "but got type: " + str(type(map_location)))
        if (str(map_location).startswith('cuda')):
            validate_cuda_device(map_location)

        cu = torch._C.CompilationUnit()
        if isinstance(f, str) or isinstance(f, pathlib.Path):
>           cpp_module = torch._C.import_ir_module(cu, f, map_location, _extra_files)
E           TypeError: import_ir_module(): incompatible function arguments. The following argument types are supported:
E               1. (arg0: torch._C.CompilationUnit, arg1: str, arg2: object, arg3: torch._C.ExtraFilesMap) -> torch._C.ScriptModule
E
E           Invoked with: <torch._C.CompilationUnit object at 0x7f523d3c1f80>, PosixPath('*****'), None, ExtraFilesMap{}

```

What is the reason of this runtime error?
In mu humble opinion, it seems that the arg1 of function `import_ir_module` has to be str, but PosixPath is passed. According to the official document of PyTorch, [it seems that the arg1 of import_ir_module only accepts string](https://pytorch.org/cppdocs/api/function_namespacetorch_1_1jit_1a5420699513343f04ec906ee559a51a1b.html).
Thanks.

**Comment by Tagussan** [comment]
I've found a workaround !! I modified the source code of `torch-neuron` directly.
On line 228 in `aws_neuron_pytorch_p36/lib/python3.6/site-packages/torch/jit/__init__.py` , changing the following line

```
cpp_module = torch._C.import_ir_module(cu, f, map_location, _extra_files)
```
into
```
cpp_module = torch._C.import_ir_module(cu, str(f), map_location, _extra_files)
```
solved the problem. As the previous line `if isinstance(f, str) or isinstance(f, pathlib.Path):` also accepts `pathlib.Path` object, the argument to import_ir_module should be casted correctly.

Because the source code of torch-neuron is not open, I cannot make a pull-request for this. However, I'd like to suggest this modification in the code.

**Comment by Tagussan** [comment]
Sorry, this is not a torch-neuron related stuff, but a original torch problem.
This problem is fixed in PyTorch v1.7.1. Easy workaround for this is to give the path of the model to `torch.jit.load` with str, not with `pathlib.Path` object.
